app/assets/stylesheets/movies.scss

```diff
@@ -30,13 +30,14 @@ article.tile-cover-pic {
   position: relative;
   background-color: $missing-poster-background-color;
   float: left;
-  margin: -10px 10px 10px 0px;
+  margin: 10px 10px 10px 0px;
```

Comment:
This css change allows the TV show name text to fit more gracefully on the "missing image" poster.